src/modules/clusters/cluster-list/cluster-list.js

```diff
@@ -100,6 +96,326 @@
             isAscending: true
         };
 
+        /*BEGIN Unmanage confirm modal*/
+        vm.showUnmanageModal = false;
+        vm.openUnmanageModal = openUnmanageModal;
+        vm.closeUnmanageModal = closeUnmanageModal;
+        vm.unmanageModalId = "unmanageModal";
+        vm.unmanageModalTitle = "Unmanage Cluster";
+        vm.unmanageTemplate = "/modules/clusters/unmanage-cluster/unmanage-confirm/unmanage-confirm.html";
+        vm.unmanageActionButtons = [{
+                label: "Cancel",
+                isCancel: true
+            },
+            {
+                label: "Unmanage",
+                class: "btn-primary custom-class",
+                actionFn: function() {
+                    vm.showUnmanageProgress = true;
+                }
+
+            }
+        ];
+
```

Comment:
@a2batic brackets should be like `}] `. Please make this change at all other places too.

Reply:
Ack